src/Symfony/Bridge/Twig/Extension/FormExtension.php

```diff
@@ -79,6 +88,74 @@ public function getTests(): array
             new TwigTest('rootform', 'Symfony\Bridge\Twig\Extension\twig_is_root_form'),
         ];
     }
+
+    public function getFieldName(FormView $view): string
+    {
+        $view->setRendered();
+
+        return $view->vars['full_name'];
+    }
+
+    public function getFieldValue(FormView $view): string
+    {
+        return $view->vars['value'];
+    }
+
+    public function getFieldLabel(FormView $view): Translatable
+    {
+        return new Translatable(
```

Comment:
when the translation component is not installed, shouldn't this return only `$view->vars['label']`? (same in the other methods)

Reply:
Actually, shouldn't the twig function always call the translator when one is passed?
I don't see the use case for getting the untranslated value in the view.

Reply:
i think generally spoken `form.vars.label` should be a translatable, always to be passed to `|trans` in the view layer.

Now with the `TranslatableInterface` an interesting usecase pops up: `label => t('translated')` vs `label => "literal"`

The string can be normalized to a `Untranslatable implements TranslatableImplemention` type, sort of speak :D to enforce a no-op.